src/app/(userFacing)/manage-reports/page.tsx

```diff
@@ -0,0 +1,108 @@
+import { CardTitle, CardHeader, CardContent, Card } from "@/components/ui/card"
+import { TableHead, TableRow, TableHeader, TableCell, TableBody, Table } from "@/components/ui/table"
+import { isAdmin } from "@/app/_actions/auth"
+import { DropdownMenu, DropdownMenuContent, DropdownMenuTrigger } from "@/components/ui/dropdown-menu"
+import { DropdownItem } from "@/components/OrdersDropdownMenu"
+import StatusBadge from "@/components/StatusBadge"
+import { FaPencilAlt } from "react-icons/fa";
+import { redirect } from "next/navigation"
+import { ReportStatus } from "@prisma/client"
+import { getAllReports } from "@/app/_actions/report"
+
+// TODO: make reused code cleaner?
+
+
+// Admin view of all reports
+export default async function Reports() {
+  const admin = isAdmin();
+  if (!admin) redirect("/");
+  const reports = await getAllReports();
+  if (!reports) throw new Error("Unknown error occured while getting reports");
```

Comment:
Maybe should remove this? cus if there is no reports initially then this would crash right